### PR TITLE
Resolve issues with quickstart for Windows users

### PR DESCRIPTION
**Pull request by nwayve**
This updates the quickstart instructions for deploying the Conduit quickstart example.
- A build.js file breaks up the build command and makes it Windows friendly for those following along the setup instructions for Windows.
- `npm` references changed to `npm.cmd` since the commands executing the runway build_steps were not finding the implicit `npm` command.
- Removed `CRLF` line endings in favor of `LF`
- Added a setting for vscode users that have reStructuredText extension for syntax highlighting.

### CODE REVIEW

File: docs/source/quickstart.rst [hunk 1]
```diff
@@ -141,8 +141,11 @@ simple demonstration of using Runway to deploy a Serverless Framework backend wi
     Remove-Item 35a66d144d8def340278cd55080d5c745714aca4.zip -Force
     Rename-Item angular-realworld-example-app-35a66d144d8def340278cd55080d5c745714aca4 frontend
     cd frontend
-    $(gc .\package.json) -replace "^\s*`"build`":\s.*$", "    `"build`": `"if test \`"`$(pipenv run runway whichenv)\`" = \`"prod\`" ; then ng build --prod --base-href .\/ && cp CNAME dist\/CNAME; else ng build --base-href .\/ && cp CNAME dist\/CNAME; fi`"," | Out-File .\package.json -Force -Encoding UTF8
+    (gc .\package.json -raw).Replace("`"rxjs`": `"^6.2.1`"", "`"rxjs`": `"~6.3.3`"") | sc .\package.json
+    Invoke-WebRequest https://raw.githubusercontent.com/onicagroup/runway/master/quickstarts/conduit/build.js -OutFile scripts/build.js
+    $(gc .\package.json) -replace "^\s*`"build`":\s.*$", "    `"build`": `"node scripts/build`"," | Out-File .\package.json -Force -Encoding UTF8
     npm install
+    mkdir scripts
```
Comment (troyready) [inline docs/source/quickstart.rst]: Can you move this above the download ^

File: docs/source/quickstart.rst [hunk 2]
```diff
@@ -141,8 +141,11 @@ simple demonstration of using Runway to deploy a Serverless Framework backend wi
     Remove-Item 35a66d144d8def340278cd55080d5c745714aca4.zip -Force
     Rename-Item angular-realworld-example-app-35a66d144d8def340278cd55080d5c745714aca4 frontend
     cd frontend
-    $(gc .\package.json) -replace "^\s*`"build`":\s.*$", "    `"build`": `"if test \`"`$(pipenv run runway whichenv)\`" = \`"prod\`" ; then ng build --prod --base-href .\/ && cp CNAME dist\/CNAME; else ng build --base-href .\/ && cp CNAME dist\/CNAME; fi`"," | Out-File .\package.json -Force -Encoding UTF8
+    (gc .\package.json -raw).Replace("`"rxjs`": `"^6.2.1`"", "`"rxjs`": `"~6.3.3`"") | sc .\package.json
+    Invoke-WebRequest https://raw.githubusercontent.com/onicagroup/runway/master/quickstarts/conduit/build.js -OutFile scripts/build.js
+    $(gc .\package.json) -replace "^\s*`"build`":\s.*$", "    `"build`": `"node scripts/build`"," | Out-File .\package.json -Force -Encoding UTF8
```
Comment (troyready) [inline docs/source/quickstart.rst]: This looks good to me -- can you change the posix example to use the build script too so they stay in sync?

Comment (nwayve) [reply docs/source/quickstart.rst]: yeah, we can do that

File: docs/source/quickstart.rst (hunk 1 above)
Comment (nwayve) [reply docs/source/quickstart.rst]: Oops, yeah, that's where that was supposed to be 😄

**Comment by nwayve** [comment]
Updated macOS/Linux instructions and fixed `mkdir scripts` ordering on Windows instructions.

**Pull request merged**